src/reason_parser.mly

```diff
@@ -1049,6 +1064,8 @@ conflicts.
 
 %nonassoc below_LBRACKETAT
 %nonassoc LBRACKETAT
+%nonassoc SLASHGREATERLESS
+%nonassoc GREATERLESS
```

Comment:
Do you understand why these precedences for these tokens must be exactly as they are specified here?